extensions/ql-vscode/package.json

```diff
@@ -1187,7 +1187,9 @@
     "watch": "npm-run-all -p watch:*",
     "watch:extension": "tsc --watch",
     "watch:webpack": "gulp watchView",
-    "test": "mocha --exit -r ts-node/register test/pure-tests/**/*.ts",
+    "test": "npm-run-all -p test:*",
+    "test:mocha": "mocha --exit -r ts-node/register test/pure-tests/**/*.ts",
```

Comment:
Rather than splitting based on the technology used, shall we split on the kinds of tests? the jest tests are currently React component tests (e.g. `test:ui` or `test:view`), and mocha tests are other tests (e.g. `unit`?)